image/state.go

```diff
@@ -0,0 +1,65 @@
+package image
+
+import (
+	"sync"
+)
+
+var (
+	iSR = &imageStateMgr{
+		imgIDMap: make(map[ID]refCount),
+	}
```

Comment:
I think the  image operation just a temporary state. for example when we save/load/rm a image only when the dockerd is working, if dockerd restarted, I think the operation state SHOULD be cleared. It is non-persistent, so I do not save it in daemon or image.Store.

How about others' opinions?